pkg/controller/pxc/secrets.go

```diff
@@ -126,6 +126,9 @@ func validatePasswords(secret *corev1.Secret) error {
 			if strings.ContainsAny(string(pass), ";:") {
 				return errors.New("invalid proxyadmin password, don't use ';' or ':'")
 			}
+			if string(pass)[0:1] == "*" {
```

Comment:
Since we are already using the `strings` from the standard library, let's do the following here

```
strings.HasPrefix(string(pass), "*")
```

Reply:
Sure! I've updated this.